### ISSUE TITLE
Show with pressure on fullscreen windows

### ISSUE THREAD

**Issue by sassanh** [micheleg/dash-to-dock#725]
Thanks for this great extension.
I have a minor problem with full screen windows. The problem is whenever I'm in a fullscreen in window I have to switch to another screen in order to be able to use the dock. When I activate intelligent autohide I can open the dock by pressing pointer to bottom edge but I don't wanna use autohide on normal screens. It would be nice if there was an option that by pressing the pointer to bottom edge, the dock would show even when the autohide is disabled.

**Comment by sassanh** [comment]
Actually by investigating more it should be as simple as having an option for not hiding the dock for windows that overlap with it in intellihide mode and just hide it for fullscreen windows.

**Comment by sassanh** [comment]
No I was wrong, in intellihide it doesn't take room and comes as an overlay which is not desired.

**Comment by micheleg** [comment]
Yes, it would be a bit complicated to mix the two behaviour given the different way in which the dock gets allocated the space, so not something trivial to implement. At the same time this would need to be optional, because I'm sure most people having intellihide don't want to have the behaviours mixed.

**Comment by EndrII** [comment]
@vanvugt This issue was created 3 years age... 
This feature is change only one line on default config in the ubuntu-dock branch... this is so hard ?
